### DISCUSSION TITLE
Is it possible to set a default path for local replica?

### DISCUSSION THREAD

**Discussion by e-zz** [overleaf-workshop/Overleaf-Workshop#137] [Q&A] [answered]
Hi, 
Since I use the local replica feature a lot, I'm wondering if there's such a default option for the local replica path.
Most of the time I just wanna open the project locally so plugins are fully enabled. It really doesn't matters so much where the replica actually is.
In theory, a local replica needs to be initiated just once, but in fact the plugin might lose track of replica so sometimes we have to create multiple times. So it will help if we could have such a default option I think. 
Thanks

**Comment by iamhyc** [comment, marked as answer]
Yes, I think we could have the **home folder** as the default path, and allow only one local replica for one project so you don't have to choose it every time.

About the out-of-sync thing, it still takes time to stabilize the feature. I will firstly fix all the matters reported in the issue now :)

**Reply by e-zz** [reply]
Great to know. Thanks a lot!
